doc/developer/scratch.md

```diff
@@ -124,5 +126,87 @@ bin/scratch destroy --all-mine
 Pass `--dry-run` if you want to see what instances `bin/scratch destroy` would
 destroy without actually destroyin them.
 
+### Grafana integration
+
+The `materialized` process always exposes metrics at via its primary port's http server on the
+prometheus-standard `/metrics` path, but the deployed server needs to be configured correctly for
+our Prometheus server to actually scrape the metrics and thereby expose them to Grafana. You
+
+The tl;dr is that you must configure mzcompose with `--preserve-ports` and the EC2 instance with
+the `"scrape_benchmark_numbers": "true"` tag. Thus a bare-minimum grafana-integrate config looks
+like:
+
+```javascript
+{
+    "launch_script": "bin/mzcompose --preserve-ports ..<remainder of args>"
+    // .. snip config ..
+    "tags": {
+        "scrape_benchmark_numbers": "true"
+    }
+}
+```
+
+Read on for more details and some other items that can be configured.
+
+#### Prometheus config
+
+There are three tags that configure our Prometheus' integration, and if you are running inside
+mzcompose it must also be configured correctly.
+
+The three Prometheus tags are:
```

Comment:
```suggestion
There are three tags on the EC2 instance that configure our Prometheus integration:
```

(I think it makes more sense not talk about mzcompose here; you've already talked about it in general in the overview section above and you've got a dedicated section for it below!)